connectors/src/connectors/google_drive/lib/permissions.ts

```diff
@@ -1,17 +1,21 @@
+import type { ContentNodesViewType } from "@dust-tt/types";
+
 import {
   isGoogleDriveFolder,
   isGoogleDriveSpreadSheetFile,
 } from "@connectors/connectors/google_drive/temporal/mime_types";
 import { getDocumentId } from "@connectors/connectors/google_drive/temporal/utils";
 import type { GoogleDriveFiles } from "@connectors/lib/models/google_drive";
 
-export function getPermissionViewType(file: GoogleDriveFiles) {
+export function getPermissionViewType(
+  file: GoogleDriveFiles,
+  viewType: ContentNodesViewType
+) {
   if (isGoogleDriveFolder(file)) {
     return "folder";
   }
-
-  if (isGoogleDriveSpreadSheetFile(file)) {
-    return "database";
+  if (isGoogleDriveSpreadSheetFile(file) && viewType === "tables") {
```

Comment:
So if the view is not `tables` we will display spreadsheets as regular file, is that expected?

Reply:
yes !